Sensus.Android/Sensus.Android.csproj

```diff
@@ -20,7 +20,7 @@
     <AndroidUseLatestPlatformSdk>false</AndroidUseLatestPlatformSdk>
     <ReleaseVersion>2.0.0</ReleaseVersion>
     <TargetFrameworkVersion>v6.0</TargetFrameworkVersion>
-    <AssemblyName>SensusAndroid</AssemblyName>
+    <AssemblyName>Sensus.Android</AssemblyName>

```

Comment:
Oh yeah, I just noticed it when verifying a few settings for XTC and assumed it was an oversight.